### PR TITLE
Drop old sqlalchemy (nine)

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

**Comment by djmitche** [comment]
@rutsky

File: builders.py [hunk 1]
```diff
@@ -450,16 +450,13 @@ def mklintyfactory():
 
 config_slaves = names(get_slaves(run_config=True, py27=True))
 
-sa0710 = 'sqlalchemy==0.7.10'
 sa087 = 'sqlalchemy==0.8.7'
```
Comment (rutsky) [inline builders.py]: Technically speaking we doesn't test with 0.8.0, but state BB should work on it. However if anyone uses 0.8.* he probable uses version close to latest patch-level release of 0.8, so this should be ok.

File: builders.py [hunk 2]
```diff
@@ -450,16 +450,13 @@ def mklintyfactory():
...
 sa099 = 'sqlalchemy==0.9.9'
 sa100 = 'sqlalchemy==1.0.0'
-sam072 = 'sqlalchemy-migrate==0.7.2'
 sam090 = 'sqlalchemy-migrate==0.9.0'
 sam098 = 'sqlalchemy-migrate==0.9.8'
 
 sqlalchemy_combos = [
-    (sa0710, sam072),  # old, deprecated in 0.9.0
```
Comment (rutsky) [inline builders.py]: Is it ok for the eight branch?

Comment (rutsky) [inline builders.py]: Sorry, I missed that this is for the nine branch only!

**Pull request merged**